src/components/cards/Cards.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import type { IProduct } from 'data/cards';
+import './cards.css';
+
+interface Props {
+  products: IProduct[];
+}
+
+export class Cards extends Component<Props, Record<string, unknown>> {
+  constructor(props: Props) {
```

Comment:
If you don’t initialize state and you don’t bind methods, you don’t need to implement a constructor for your React component.

https://legacy.reactjs.org/docs/react-component.html#constructor